docs/src/pages/en/media-slot.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Media slot
+description: Understand how the media slot and the media element works in Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+The key insight with Media Chrome is separating the UI from the media playback. Media Chrome is agnostic to whatever playback engine you are using to play your media. Media Chrome is purely about building the UI.
+
+Media Chrome uses the media slot specified with `slot="media"` in order to interact with the media. For example, when a viewer clicks the "play" button, Media Chrome will call `play()` on whatever element is specified in the media slot.
+
+<h2>Media Slot</h2>
+
+The simplest media slot is the browser's native `<video>` tag. For example:
+
+```html
+<media-controller>
+  <video slot="media" src="https://....mp4" >
+</media-controller>
+```
+
+In this example, Media controller will:
+
+* Listen for events emitted from the `<video>` element
+* Understand the current state of the `<video>` element by using the known props
+* Call methods on the `<video>` element like `play()`, `pause()`, etc.
+
+The superpower of Media Chrome is that the media slot does not have to be a `<video>` element. It can be any HTML element that [implements the HTML5 Video Element API](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video).
```

Comment:
Can we provide examples of other HTML elements that implement this API?

Reply:
that's in the next section under `Media Elements`, could that be more clear?

Reply:
Ah I didn't make the connection initially. Since a new h2 follows this paragraph, it felt like a new thought. Maybe it's clear after all, or we could add a "Read on for some examples" or "Here are a few examples:" something to that effect?